csun_paper/aom-csun-2018-extended-abstract.md

```diff
@@ -0,0 +1,272 @@
+an HTML element using JavaScript instead of by setting an attribute.
+For example, here's the HTML for an ARIA checked checkbox.
+
+```html
+<div role="checkbox" aria-checked="true">...</div>
+```
+
+Instead, it'd be possible to write the following code:
+
+```js
+element.accessibleNode.role = "checkbox";
+element.accessibleNode.checked = true;
+```
+
+One use case this solves is that Custom Elements [3] "sprout"
+attributes in order to express their own semantics. AOM allows
+custom elements to behave more like native HTML elements, where
+their accessibility behavior is encapsulated.
+
+Furthermore, writing Accessible Properties would allow specifying
+accessible relationships without requiring IDREFs, as authors can now
+pass object references. Currently attributes like aria-describedby
+and aria-activedescendant take one or more IDREFs, like this:
+
+```html
+<div role="listbox" aria-activedescendant="id3">
+```
+
+Accessible Properties would allow passing object references instead:
+
+```js
+element.accessibleNode.activeDescendant = accessibleNode3;
+```
+
+Moreover, writing Accessible Properties would enable authors using Shadow DOM
+to specify relationships which cross over Shadow DOM boundaries.
+
+While AOM and ARIA both affect the computed accessible properties of a
+node, and have the same vocabulary, they are separate interfaces. ARIA
+does not reflect to AOM. In the case where and AOM Accessible Property
+and the corresponding ARIA attribute have different values, the AOM
+property takes precedence.
+
+### Phase 2: Accessible Actions
+
+**Accessible Actions** will allow authors to react to user input
+  events coming from assistive technology.
+
+For example, a mobile screen reader user may navigate to a slider,
+then perform a gesture to increment a range-based control.  The screen
+reader sends the browser an increment *action* on the slider element
+in the accessibility tree.
+
+Currently, browsers implement partial support by accessible actions either by
+implementing built-in support for native HTML elements (for example, a native
+HTML `<input type="range">` already supports increment and decrement actions,
+and text boxes already support actions to set the value or insert text.
+
+However, there is no way for web authors to listen to accessible actions on
+custom elements. A custom slider won't react to increment and decrement
+gestures.
+
+Accessible Actions gives web developers a mechanism to listen for accessible
+actions directly, by adding event listeners on an `AccessibleNode`.
+This is analogous to listening for user interaction events on a DOM node,
+except that the interaction event arrives via an assistive technology API,
+so it is directed to the accessible node first.
+
+To implement a custom slider, the author could simply listen for
+`increment` and `decrement` events.
+
+```js
+customSlider.accessibleNode.addEventListener('accessibleincrement', function() {
```

Comment:
I sort of dislike to reuse DOM events API for AOM, but that's a different topic for sure.